src/components/Dashboard/StaffNotation.vue

```diff
@@ -1,74 +1,78 @@
-        />
-        <AddStaffNotationDialog
-            :dialog="isAddingRegistrarsOrder"
-            @close="hideRegistrarsOrderDialog($event)"
-            attach="#staff-notation"
-            displayName="Registrar's Order"
-            name="registrarsOrder"
-        />
-        <AddStaffNotationDialog
-            :dialog="isAddingCourtOrder"
-            @close="hideCourtOrderDialog($event)"
-            attach="#staff-notation"
-            displayName="Court Order"
-            name="courtOrder"
-            courtOrderNumberRequired="true"
-        />
-        <div class="filing-item__actions">
-            <v-menu offset-y left transition="slide-y-transition" v-model="expand">
-                <template v-slot:activator="{ on }">
-                    <span><!-- This span is needed to fix a positioning issue with the menu -->
-                        <span
-                            v-on="on"
-                            id="add-staff-filing-label"
-                            class="app-blue"
-                            @click="expand = !expand"
-                        >
-                            <v-icon id="add-staff-filing-icon" class="app-blue">mdi-plus</v-icon>
-                            <span>Add Staff Filing</span>
-                        </span>
-                        <v-btn
-                            text
-                            v-on="on"
-                            class="menu-btn"
-                            :class="{active: expand}"
-                            @click="expand = !expand"
-                        >
-                            <v-icon>mdi-menu-down</v-icon>
-                        </v-btn>
-                    </span>
-                </template>
-                <v-list dense>
-                    <v-list-item-group color="primary">
-                        <v-list-item @click="showRegistrarsNotationDialog()" :disabled="disabled">
-                            <v-list-item-title>
-                                <span class="app-blue">Add Registrar's Notation</span>
-                            </v-list-item-title>
-                        </v-list-item>
-                        <v-list-item @click="showRegistrarsOrderDialog()" :disabled="disabled">
-                            <v-list-item-title>
-                                <span class="app-blue">Add Registrar's Order</span>
-                            </v-list-item-title>
-                        </v-list-item>
-                        <v-list-item @click="showCourtOrderDialog()" :disabled="disabled">
-                            <v-list-item-title>
-                                <span class="app-blue">Add Court Order</span>
-                            </v-list-item-title>
-                        </v-list-item>
-                    </v-list-item-group>
-                </v-list>
-            </v-menu>
-        </div>
+  <div id="staff-notation" class="pr-6" :class="{ 'add-scrollbar-offset': addScrollbarOffset }">
```

Comment:
OK, _mostly_ I just re-indented this file. Hide whitespace changes to see this more cleanly.